events/event_nettrace.go

```diff
@@ -16,6 +20,18 @@ var NetTraceIntegrator = func() func(basictracer.SpanEvent) {
 			tr.SetMaxEvents(1000)
 		case basictracer.EventFinish:
 			tr.Finish()
+		case basictracer.EventTag:
+			tr.LazyPrintf("%s:%v", t.Key, t.Value)
+		case basictracer.EventLogFields:
+			var buf bytes.Buffer
+			for i, f := range t.Fields {
+				if i > 0 {
+					buf.WriteByte(' ')
+				}
+				fmt.Fprintf(&buf, "%s:%v", f.Key(), f.Value())
+			}
+
+			tr.LazyPrintf("%s", buf.String())
```

Comment:
Post-merge nit-picking.
tr.LazyPrintf("%s", x)
is perhaps less desirable than
tr.LazyLog(x, false)
and since bytes.Buffer is a fmt.Stringer itself, using LazyLog would defer the []byte to string conversion here.